homeassistant/components/plex/cast.py

```diff
@@ -0,0 +1,70 @@
+"""Google Cast support for the Plex component."""
+from __future__ import annotations
+
+from pychromecast import Chromecast
+from pychromecast.controllers.plex import PlexController
+
+from homeassistant.components.cast.const import DOMAIN as CAST_DOMAIN
+from homeassistant.components.media_player import BrowseMedia
+from homeassistant.components.media_player.const import MEDIA_CLASS_APP
+from homeassistant.core import HomeAssistant
+
+from . import async_browse_media as async_browse_plex_media, is_plex_media_id
+from .const import PLEX_URI_SCHEME
+from .services import lookup_plex_media
+
+
+async def async_get_media_browser_root_object() -> BrowseMedia:
```

Comment:
Note that there could be multiple root objects (ie multiple config entries)

Reply:
Fixed in 10ffd80